app/route_rangers_api/static/graphs.js

```diff
@@ -0,0 +1,175 @@
+function drawgraph(csv,xaxis,yaxis) {
+    var margin = {top: 30, right: 30, bottom: 70, left: 60},
```

Comment:
should we use the same margins for all the maps? We should maybe move this out of the function

Reply:
We can standardize but its really hard to get a JS function to reference a style sheet so i tried to put it in the function. @meganhmoore any chance this is something you know how to do? Or I'll ask james on friday. But happy to standardize across graphs/ maps

Reply:
yeah I'd be happy to discuss this in detail tomorrow but essentially my hope is that we can scale things such that we have the majority of the styling in the html that encompasses the components (like the styling associated with each class for each div that sets their height and width in a responsive way `class="col-12 col-md-4 col-lg-4"`) and then we do very minimal styling in the javascript itself. 

Additionally in general when setting widths, margins, etc it would probably be best to do it in terms of percentages (5% margin rather than 5px) because it will make it much more standardizable.